test/metrics_test.py

```diff
@@ -1,17 +0,0 @@
-from . import ApolloTestCase, wa
```

Comment:
well, the original reason was because I'm also using this to switch the backend between grails 2 to grails 4 and I wasn't adding the same metric library.  

That being said, I think that it might make more sense for me to put a placeholder in there. . . . I'll add it back, but then verify the same on the Apollo side.   Should be ready by tomorrow.

Reply:
it is added back